docs/decisions/003-typescript-only.md

```diff
@@ -0,0 +1,30 @@
+# TypeScript Only
+
+Date: 2024-01-26
```

Comment:
we probably decided for this much earlier

Reply:
I know, but I added the date of the creation of the ADR, not the decision itself.